packages/synapse-react-client/src/components/SynapseTable/SynapseTable.tsx

```diff
@@ -258,6 +258,7 @@ export function SynapseTable(props: SynapseTableProps) {
                         style={{
                           width: cell.column.getSize(),
                         }}
+                        align={(cell.column.columnDef.meta as any)?.align}
```

Comment:
Read column alignment from the `meta` field, which can contain any column-related data we want to specify, and apply it to the `td` element

Reply:
Useful and generalizable, I like it